avaframe/com1DFA/com1DFA.py

```diff
@@ -2534,6 +2576,74 @@ def exportFields(cfg, Tsave, fieldsList, dem, outDir, logName):
         countTime = countTime + 1
 
 
+def exportFields1(cfg, timeStep, fields, dem, outDir, logName, TSave="intermediate", resTypesForced=[]):
```

Comment:
See above about the naming. Please modify the original one, so the differences can be seen